### PR TITLE
add a new opp job on OCP 4.14 that runs weekly on wednesdays

### PR DESCRIPTION
**Pull request by vboulos**
This PR adds a new OPP job with OCP 4.14, and executes weekly on Wednesdays

### CODE REVIEW

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Review by gparvin — APPROVED** [review]

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by gparvin** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by gparvin** [comment]
PR https://github.com/open-cluster-management-io/policy-collection/pull/432 was merged which should address the issue encountered in the rehearsal run where the machines could not be provisioned due to no AMI ID for OCP 4.14 being defined in the setup policy set.

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws

**Review by gparvin — APPROVED** [review]

File: ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml [hunk 1]
```diff
@@ -92,3 +92,85 @@ periodics:
...
+    org: stolostron
+    repo: policy-collection
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-interop-qe
+    ci-operator.openshift.io/variant: opp-ocp4.14-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-stolostron-policy-collection-main-opp-ocp4.14-interop-openshift-plus-interop-aws
```
Comment (ascerra) [inline ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml:113]: Add slack notification section under this line if desired

```
reporter_config:
    slack:
      channel: '#opp-discussion'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'
```

**Pull request merged**